### PR TITLE
Implement GQL create and delete for OCR key bundles

### PR DESCRIPTION
**Pull request by makramkd**
Ticket: https://app.shortcut.com/chainlinklabs/story/20537/add-gql-endpoints-for-ocr-keys

### CODE REVIEW

File: core/web/schema/type/ocr.graphql [hunk 1]
```diff
@@ -8,3 +8,14 @@ type OCRKeyBundle {
...
+type OCRKeyBundlePayload {
+    bundle: OCRKeyBundle!
+}
+
+type OCRError implements Error {
+    message: String!
+	code: ErrorCode!
+}
+
+union DeleteOCRKeyBundlePayload = OCRKeyBundlePayload | OCRError
```
Comment (jkongie) [inline core/web/schema/type/ocr.graphql:21]: Just make this a `NotFoundError`

```suggestion
union DeleteOCRKeyBundlePayload = OCRKeyBundlePayload | NotFoundError
```

**Review by jkongie — CHANGES_REQUESTED** [review]

File: core/web/resolver/mutation.go [hunk 2]
```diff
@@ -260,3 +262,34 @@ func (r *Resolver) UpdateFeedsManager(ctx context.Context, args struct {
...
+}) (*DeleteOCRKeyBundlePayloadResolver, error) {
+	if err := authenticateUser(ctx); err != nil {
+		return nil, err
+	}
+
+	deletedKey, err := r.App.GetKeyStore().OCR().Delete(args.ID)
+	if err != nil {
+		if errors.As(err, &keystore.KeyNotFoundError{}) {
+			return NewDeleteOCRKeyBundlePayloadResolver(ocrkey.KeyV2{}, err), nil
+		}
```
Comment (makramkd) [inline core/web/resolver/mutation.go:290]: Let me know your thoughts on this as opposed to e.g `strings.HasPrefix` on `err.Error()`.

File: core/services/keystore/ocr.go [hunk 3]
```diff
@@ -157,7 +167,7 @@ func (ks *ocr) GetV1KeysAsV2() (keys []ocrkey.KeyV2, _ error) {
 func (ks *ocr) getByID(id string) (ocrkey.KeyV2, error) {
 	key, found := ks.keyRing.OCR[id]
 	if !found {
-		return ocrkey.KeyV2{}, fmt.Errorf("unable to find OCR key with id %s", id)
+		return ocrkey.KeyV2{}, KeyNotFoundError{id}
```
Comment (jkongie) [inline core/services/keystore/ocr.go:170]: Great!

File: core/web/resolver/mutation.go (hunk 2 above)
Comment (jkongie) [reply core/web/resolver/mutation.go:290]: This is much better

**Review by jkongie — APPROVED** [review]

**Pull request merged**